src/main/webapp/step3-startup.js

```diff
@@ -0,0 +1,36 @@
+import Step3Renderer from './renderers/Step3Renderer.js';
+
+// TODO: replace this array with the event management class, as discussed with Remus
+const events = [{
+        name: 'Breakfast',
+        location: 'Zurich',
+        startingTime: '9:30',
```

Comment:
Times are strings?

Reply:
I am not sure whether we should convert them to Date or maybe create a custom Time class that accepts only hour and minute (trade-off single day events vs events that span multiple days). Need to have a brief discussion about this!

Reply:
I conducted some research and it seems that the directions service accepts only Date, so Date is the best option in our case!